input/pagecontent/2-3-Events.md

```diff
@@ -106,7 +106,7 @@ SelectionEventName ::= ( FHIRresource  ) '-' ( 'select' )
 
 {% include img.html img="SelectionEventName.png" caption="Figure: Selection Event-name specification" %}
 
-The `FHIRresource` indicates the context of the selection. The `context` element in a select event typically contains two fields. One with the name of the `FHIRresource` holding the anchor resource and one or more named `select` indicating the resources that are selected.
+The `FHIRresource` indicates the context of the selection. The `context` element in a select event typically contains two fields. One field with the name of the fhir-resource holding the anchor resource, and one or more fields named select indicating the resources that is selected.

```

Comment:
I am not sure why to change from FHIRresource to fhir-resource but am OK if folks want to do so.  FHIRresource is consistent with the EBNF diagram and other naming in the text of event descriptions.  If the intent is simply to "generically" refer to a FHIR resource maybe FHIR resource is better than fhir-resource?

Reply:
Eric, I was sloppy with this change and just copied and pasting the jira commenter's proposed wording, despite us having made additional changes to the sentence since his comment was written. I took almost all of your suggestion.